asserts/database.go

```diff
@@ -61,6 +60,20 @@ type Backstore interface {
 	SearchBySuffix(assertType AssertionType, primaryPathWithoutLast []string, suffixOflast string, foundCb func(Assertion)) error
 }
 
+// KeypairManager is a manager and backstore for private/public key pairs.
+type KeypairManager interface {
+	// Import stores the given private/public key pair for identity and
+	// returns its fingerprint
+	Import(authorityID string, privKey PrivateKey) (fingerprint string, err error)
+	// Get returns the private/public key pair with the given fingeprint.
+	Get(authorityID, fingeprint string) (PrivateKey, error)
```

Comment:
@niemeyer yes, I'm working on the branch to add explict keyids, my idea was to look again at this after we have explict keyids around